static/app/utils/queryString.tsx

```diff
@@ -4,6 +4,10 @@ import {escapeDoubleQuotes} from 'sentry/utils';
 import type {Sort} from 'sentry/utils/discover/fields';
 import {safeURL} from 'sentry/utils/url/safeURL';
 
+// Create a string representation of the regex so we need to create a new regex
+// for each use to avoid carrying over state
+export const TAG_VALUE_ESCAPE_PATTERN = '[:\\s\\(\\)\\\\"]';
```

Comment:
Converting the old regex to the new one required us to double up on escaping backslashes. It's not the most ideal but we do this because `.test()` actually stores some state so I can't just export this as a simple regex.